.idea/prettier.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
**.idea** should be ignored in VCS. To do that, include it it **.gitignore** file at root.